main/integration-tests/__test__/api-tests/aws-accounts/get-aws-account.test.js

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('Get AWS Account scenarios', () => {
+  let setup;
+  let adminSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Getting an AWS Account', () => {
+    it('should fail if user is inactive', async () => {
+      const admin2Session = await setup.createAdminSession();
+      await adminSession.resources.users.deactivateUser(admin2Session.user);
+
+      await expect(
+        admin2Session.resources.awsAccounts.awsAccount(setup.defaultAwsAccountId).get(),
```

Comment:
If we are starting this new pattern, can we have it setup.defaults.awsAccountId?  this way we can add more defaults in the future and also allow components/addons to contribute.  We can do something similar to getServices() getGenerators(), etc. So we can have getDefaults().  What do you think?

Reply:
As discussed, I'll cover the `defaults` pattern in my next resource test suite development.